components/01-atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,86 @@
+$button-radii: (
+  'soft': var(--button-radius-soft),
+  'pill': 100vw,
+);
+$button-style: (
```

Comment:
I don't see where these `$button-style` value are being used? Removing them had no affect that I could see. If these are needed I would think they should use the `$size-spacing-5` and `$size-spacing-7` or some re-assignment of these tokens.

Reply:
Nope. I think I was going to, but when you look lower, there are enough fiddly differences that I couldn't do a simple @each like the radii and weights.

I've deleted it.